src/main/webapp/app/quiz/participate/quiz-scoring-infostudent-modal/quiz-scoring-info-student-modal.component.ts

```diff
@@ -0,0 +1,46 @@
+import { Component, OnInit, Input } from '@angular/core';
+import { NgbModal } from '@ng-bootstrap/ng-bootstrap';
+import { Question, QuestionType, ScoringType } from 'app/entities/question';
+
+@Component({
+    selector: 'jhi-quiz-scoring-infostudent-modal',
+    templateUrl: './quiz-scoring-info-student-modal.component.html',
+    styles: []
+})
+export class QuizScoringInfoStudentModalComponent implements OnInit {
+    readonly DRAG_AND_DROP = QuestionType.DRAG_AND_DROP;
+    readonly MULTIPLE_CHOICE = QuestionType.MULTIPLE_CHOICE;
+
+    readonly ALL_OR_NOTHING = ScoringType.ALL_OR_NOTHING;
+    readonly PROPORTIONAL_WITH_PENALTY = ScoringType.PROPORTIONAL_WITH_PENALTY;
+
+    @Input() score: number;
+    @Input() selectedOption: boolean;
+
+    @Input() correctlyChosenAnswers: number;
+    @Input() correctAnswer: number;
+    @Input() wronglyChosenAnswers: number;
+
+    @Input() amountOfAnswerOptions: number;
+
+    @Input() question: Question;
+
+    forgottenRightAnswers: number;
+
+    constructor(private modalService: NgbModal) {
+    }
+
+    ngOnInit() {
+        this.count()
+    }
+
+    open(content: any) {
+        this.modalService.open(content, { size: 'lg' });
+    }
+
+    count() {
+        this.forgottenRightAnswers = this.correctAnswer - this.correctlyChosenAnswers;
```

Comment:
I don't think this calculation is correct. Remember there are 4 cases, not only 3